selecting_reference_data.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.                distance = np.linalg.norm(row.values - centroid) # Find euclidian distance from each observation to the centroid
</pre><p>One idea for a plot - a scatterplot of the centroid of the cluster, the cluster, and the closest k points to that center point. Which was maybe already done in a development notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
How many of these plots would be created? This process is being done on all 48 weeks of a given year for each phenophase. Thus, running the function once would result in 480 plots. I think this process would be interesting to look at for just a couple of these iterations. Maybe this is something we can add to future work.

Reply:
Yes, could definitely be added to future work. It could be ran on a good sampling of weeks through the year (maybe once every 5-10 weeks), for all the phenophases for the top 10 most common species in Kerala.